### DISCUSSION TITLE
what is the difference between tauri and electronjs?

### DISCUSSION THREAD

**Discussion by ghost** [tauri-apps/tauri#6398] [Q&A] [answered]
Hi.

I would like to know where I can find some document that compares tauri and electronjs. what is the difference between tauri and electronjs? Is there any doc that compares tauri and electronjs?

**Comment by Phoeenix05** [comment, marked as answer]
I don't know if you still need help on this but this should give you some idea about what is the difference between tauri and electronjs.

This shows difference between web desktop framework differences such as build times, bundle sizes and supported platforms.
https://github.com/Elanis/web-to-desktop-framework-comparison

Also using tauri might require you to learn rust and javascript while electron only uses javascript.

Also you can just google something like "tauri vs eletronjs" or "difference between tauri and electronjs".

**Comment by shree-bitu** [comment]
Tauri is designed to be more lightweight and faster than Electron, and one more fun fact , Size of application built with Electron is typically around **~120 MB**. and the size of a Tauri app can be less than **600KB.**

**Comment by olexiyb** [comment]
You can also take a look the list of Rust GUI libraries
https://monadical.com/posts/shades-of-rust-gui-library-list.html#2-Tauri
